server/tests/models/user.spec.js

```diff
@@ -0,0 +1,191 @@
+import chai from 'chai';
+
+import chaiHttp from 'chai-http';
+import models from '../../models';
+import userTest from '../fixtures/user';
+
+chai.use(chaiHttp);
+
+const { expect } = chai;
+const { User, sequelize } = models;
+
+before(async () => {
+  await sequelize.sync({ force: true });
+});
+
+const modelTest = describe('Model - User', () => {
+  context('create()', () => {
```

Comment:
please remove context, use this as a guide https://github.com/mawrkus/js-unit-testing-guide